src/components/geolocation/GeolocationDirective.js

```diff
@@ -43,8 +43,10 @@ goog.require('ga_throttle_service');
         var view = map.getView();
         var accuracyFeature = new ol.Feature();
         var positionFeature = new ol.Feature(new ol.geom.Point([0, 0]));
-        var featuresOverlay = new ol.FeatureOverlay({
-          features: [accuracyFeature, positionFeature],
+        var featuresOverlay = new ol.layer.Vector({
+          source: new ol.source.Vector({
+            features: [accuracyFeature, positionFeature]
+          }),
           style: gaStyleFactory.getStyleFunction('geolocation')
         });
```

Comment:
See https://github.com/openlayers/ol3/blob/85815a32a073c30dc3b7aa57fb897401e3d58767/src/ol/interaction/selectinteraction.js#L177-L186 for an example.